### PR TITLE
Add link to microtask_guide

### PR DESCRIPTION
**Pull request by jchimene**
ref #1584

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/promise/index.html [hunk 1]
```diff
@@ -211,7 +211,7 @@ <h2 id="Static_methods">Static methods</h2>
 </dl>
 
 <h2 id="Instance_methods">Instance methods</h2>
-
+<p>See the <a target="_blank" href="/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide" title="Microtask_guide">Microtask guide</a> to learn more about how these methods use the Microtask queue and services. </p>
```
Comment (sideshowbarker) [inline files/en-us/web/javascript/reference/global_objects/promise/index.html:214]:
```suggestion
<p>See the <a target="_blank" href="/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide" title="Microtask_guide">Microtask guide</a> to learn more about how these methods use the Microtask queue and services.</p>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Comment by sideshowbarker** [comment]
The link is a good addition but to me at least it doesn’t seem sufficient for resolving the issue described in https://github.com/mdn/content/issues/1584:

> It should be clearly mentioned that then and catch call adds Microtask to Microtask Queue.

That is, it seems like @krp12345 is saying the above should be explained in the prose of the article.

**Comment by jchimene** [comment]
> The link is a good addition but to me at least it doesn’t seem sufficient for resolving the issue described in #1584:
> 
> > It should be clearly mentioned that then and catch call adds Microtask to Microtask Queue.
> 
> That is, it seems like @krp12345 is saying the above should be explained in the prose of the article.

That's a fair point. I'm not sure how to do that in the context of this section framework (dl,dt,dd) without duplicating the `microtask_guide`. It's difficult for me to see how to further describe how then/catch/finally enqueue Microtasks without also describing the Event Loop, et. al., which seems to be the purpose of the `microtask_guide` page. 
How about "These methods generate entries on the Microtask queue, as opposed to the Task queue. The declared functions are called during the Microtask processing loop. See <a> for more."

**Review by sideshowbarker — APPROVED** [review]

**Comment by sideshowbarker** [comment]
Sorry for not having gotten back to this one before now

**Comment by krp12345** [comment]
Thanks @sideshowbarker  and @jchimene . I am closing the issue.

**Pull request merged**